### PR TITLE
Update build.zig for upstream API changes

### PR DESCRIPTION
**Pull request by linusg**
See:
- https://github.com/ziglang/zig/pull/19167
- https://github.com/ziglang/zig/pull/19597

### CODE REVIEW

File: build.zig [hunk 1]
```diff
@@ -526,9 +525,6 @@ fn addTest(b: *std.Build, lib: *std.Build.Step.Compile,
 fn installHeader(b: *std.Build, lib: *std.Build.Step.Compile,
                  hfile: []const u8) void {
    const inc_path = "include/";
```
Comment (ivmai) [inline build.zig:527]: Is my understanding correct that '/' after "include" is redundant?

**Comment by ivmai** [comment]
Merged. Thank you!

**Comment by plajjan** [comment]
Cool!

@linusg are you using bdwgc with zig?

File: build.zig (hunk 1 above)
Comment (linusg) [reply build.zig:527]: Yes, the way you simplified it in the end should be sufficient :)

**Comment by linusg** [comment]
Thanks! @plajjan yes, I use it heavily in [my js engine](https://codeberg.org/kiesel-js/kiesel) :)

**Comment by ivmai** [comment]
> Thanks! @plajjan yes, I use it heavily in [my js engine](https://codeberg.org/kiesel-js/kiesel) :)

Cool! Before it I knew only 1 JS engine using bdwgc (but written in C++) - [Escargot](https://github.com/Samsung/escargot)

**Comment by plajjan** [comment]
Ahh nice! I'm pleased to hear the build.zig is useful to others (I did some of the initial work there) and it's great that we're a group of people that can help maintain this :)

**Comment by linusg** [comment]
I used to maintain my own barebones build.zig integration (https://codeberg.org/linusg/zig-build-libgc) but it's much nicer to have it upstream and with all build options supported - thanks for doing this work!

**Pull request closed without merging**